build.sbt

```diff
@@ -287,7 +287,7 @@ val circeV                  = "0.14.14"
 val circeGenericExtrasV     = "0.14.4"
 val circeYamlV              = "0.15.2"     // 0.15.3 drops Scala 2.12
 val jwtCirceV               = "11.0.2"
-val jacksonV                = "2.18.2"     // 2.18.3+ breaks ContentTypesSchemas.schemaForPlain
```

Comment:
Hmm there is 2.19.2 version and 2.18.4.1 (I don't know what it means..),  https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-core. Why not to upgrade to newest version?

Reply:
* 2.18.4.1 - that would require a few changes in build.sbt, because that version exists only for jackson-core
* 2.19 - I want to update one major version at a time, in case I break something that isn't covered by tests